apps/meteor/ee/app/livechat-enterprise/server/services/omnichannel.internalService.ts

```diff
@@ -23,13 +24,20 @@ export class OmnichannelEE extends ServiceClassInternal implements IOmnichannelE
 	}
 
 	async placeRoomOnHold(
-		room: Pick<IOmnichannelRoom, '_id' | 't' | 'open' | 'onHold'>,
+		room: Pick<IOmnichannelRoom, '_id' | 't' | 'open' | 'onHold' | 'u' | 'lastMessage'>,
```

Comment:
Changing the types here means you have to change the functions calling this one too. For example, `apps/meteor/ee/app/livechat-enterprise/server/api/rooms.ts L20` doesn't include `u` on its projection, so it won't be here, even if the types say so